docs/standard/parallel-programming/exception-handling-task-parallel-library.md

```diff
@@ -83,7 +83,16 @@ If a task completes in the <xref:System.Threading.Tasks.TaskStatus.Faulted?displ
 [!code-csharp[TPL_Exceptions#27](../../../samples/snippets/csharp/VS_Snippets_Misc/tpl_exceptions/cs/exceptionprop21.cs#27)]
 [!code-vb[TPL_Exceptions#27](../../../samples/snippets/visualbasic/VS_Snippets_Misc/tpl_exceptions/vb/exceptionprop21.vb#27)]
 
-In a real application, the continuation delegate could log detailed information about the exception and possibly spawn new tasks to recover from the exception.
+In a real application, the continuation delegate could log detailed information about the exception and possibly spawn new tasks to recover from the exception. There are several ways to observe exceptions directly or indirectly through propagation.
+
+| Observe                         | Description                                                                                                                 |
```

Comment:
To me, that table contains a lot of duplicated information and not related to the topic information. Also, it's not clear which ways are "direct" and which are not.

Would it be clearer to say:

> The following expressions throw an exception, if `task` faults due to an exception:
>
> - `await task`
> - `task.Wait()`
> ...and so on

After that it's worth to mention `task.Exception`.